app/assets/stylesheets/cart.css.scss

```diff
@@ -88,3 +88,23 @@
   a:visited {color: #008000; text-decoration: underline; }
 }
 
+input[type="submit"].cart-item-quantity-button {
+  border : solid 0px #ffffff;
+  border-radius : .95em;
+  moz-border-radius : 3px;
+  -webkit-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+  -moz-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+  box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+  font-size : 1em;
+  color : #ffffff;
+  padding : .4em 0;
```

Comment:
`.4` should be written with a leading zero as `0.4`<br>Property name should be immediately followed by a colon